src/options/options.html

```diff
@@ -131,6 +131,30 @@
           </div>
         </p>
       </p>
+      <p>
+        <div class="sectionTitle">Type of data connection</div>
+        <p>
+          <div>
+            <p>
+              <input type="radio" id="notifyConnectionConnectionless" name="notifyConnection" value="connectionless" checked="checked">
+              <label for="notifyConnectionConnectionless">
+                <strong>Connectionless</strong>
+                <br/>
+                A new instance of the native appliation is created for each message.
+              </label>
+            </p>
+            <p>
+              <input type="radio" id="notifyConnectionConnectionbased" name="notifyConnection" value="connectionbased">
+              <label for="notifyConnectionConnectionbased">
+                <strong>Connection based</strong>
+                <br/>
+                This launches the native application, if it is not already running.
+                The application stays running until Thunderbird is closed.
```

Comment:
Can you write `script/application` instead?

Reply:
Done in last commit.